mappings/net/minecraft/client/font/TextRenderer.mapping

```diff
@@ -163,7 +163,35 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 8 seeThrough
 		ARG 9 backgroundColor
 		ARG 10 light
-		ARG 11 rightToLeft
+	METHOD method_27517 drawWithShadow (Lnet/minecraft/class_4587;Lnet/minecraft/class_2561;FFI)I
```

Comment:
```suggestion
		ARG 11 rightToLeft
	METHOD method_27517 drawWithShadow (Lnet/minecraft/class_4587;Lnet/minecraft/class_2561;FFI)I
```

Reply:
??

Reply:
Restore the ARG 11 or the previous method